modules/codegen/src/main/scala/com/twilio/guardrail/generators/EndpointsClientGenerator.scala

```diff
@@ -511,18 +510,5 @@ object EndpointsClientGenerator {
           """
       Target.pure(NonEmptyList(Left(algebra), Right(client) :: Nil))
     }
-
-    def apply[T](term: ClientTerm[ScalaLanguage, T]): Target[T] = term match {
-      case GenerateClientOperation(className, route, methodName, tracing, parameters, responses, securitySchemes) =>
-        generateClientOperation(className, tracing, securitySchemes, parameters)(route, methodName, responses)
-      case GetImports(tracing)                                                      => getImports(tracing)
-      case GetExtraImports(tracing)                                                 => getExtraImports(tracing)
-      case ClientClsArgs(tracingName, serverUrls, tracing)                          => clientClsArgs(tracingName, serverUrls, tracing)
-      case GenerateResponseDefinitions(operationId, responses, protocolElems)       => generateResponseDefinitions(operationId, responses, protocolElems)
-      case GenerateSupportDefinitions(tracing, securitySchemes)                     => generateSupportDefinitions(tracing, securitySchemes)
-      case BuildStaticDefns(clientName, tracingName, serverUrls, ctorArgs, tracing) => buildStaticDefns(clientName, tracingName, serverUrls, ctorArgs, tracing)
-      case BuildClient(clientName, tracingName, serverUrls, basePath, ctorArgs, clientCalls, supportDefinitions, tracing) =>
-        buildClient(clientName, tracingName, serverUrls, basePath, ctorArgs, clientCalls, supportDefinitions, tracing)
-    }
```

Comment:
This is just delightful validation of the conversion.  The old interpreter just called out to methods that already existed, with the correct names and signatures, so the only change is to remove the old interpreter!

Reply:
Yeah, I was _really_ happy to see that 😄